test/jdk/jdk/jfr/jcmd/TestJcmdDump.java

```diff
@@ -105,8 +107,15 @@ private static void jfrDump(Boolean pathToGCRoots, String name, boolean disk, Pr
             leakList.add(new Object[1000_0000]);
             System.gc(); // This will shorten time for object to be emitted.
             File recording = new File("TestJCMdDump.jfr");
-            String[] params = buildParameters(pathToGCRoots, name, recording);
-            OutputAnalyzer output = JcmdHelper.jcmd(params);
+            List<String> params = buildParameters(pathToGCRoots, name, recording);
+            System.out.println(params);
+            OutputAnalyzer output = ProcessTools.executeProcess(new ProcessBuilder(params));
+            System.out.println("---------------- stdout ----------------");
+            System.out.println(output.getStdout());
+            System.out.println("---------------- stderr ----------------");
+            System.out.println(output.getStderr());
+            System.out.println("----------------------------------------");
+            System.out.println();
```

Comment:
You can just call `output.reportDiagnosticSummary()` to get this.

Reply:
Updated to use output.reportDiagnosticSummary() instead.